src/App.js

```diff
@@ -5,27 +5,19 @@ import DopplerIntlProvider from './DopplerIntlProvider';
 import { FormattedMessage } from 'react-intl';
 import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
-import axios from 'axios';
-import { HttpDopplerLegacyClient } from './services/doppler-legacy-client';
-import { OnlineSessionManager } from './services/session-manager';
 
 class App extends Component {
-  constructor(props) {
-    super(props);
+  constructor({ locale, dependencies: { sessionManager } }) {
+    super();
```

Comment:
Here I am not calling _super(props)_, only _super()_. It is discouraged in React documentation, but they do not say the reason. 

I imagine that it is because, in this way, _this.props_ is not available inside the constructor, but it is after that.

In my opinion, using deconstructing in constructor parameter make it more explicit and clear, so, I prefer this way.

Reply:
I agree it's more readable this way. `constructor({ locale, dependencies: { sessionManager } }) {`